src/lando/api/legacy/transplants.py

```diff
@@ -376,23 +376,23 @@ def warning_unresolved_comments(*, phab, revision, **kwargs):
         return "Revision has unresolved comments."
 
 
-def user_block_no_auth0_email(*, auth0_user, **kwargs):
+def user_block_no_auth0_email(*, lando_user, **kwargs):
     """Check the user has a proper auth0 email."""
     return (
         None
-        if auth0_user.email
+        if lando_user.email
         else ("You do not have a Mozilla verified email address.")
     )
 
 
-def user_block_scm_level(*, auth0_user, landing_repo, **kwargs):
+def user_block_scm_level(*, lando_user, landing_repo, **kwargs):
     """Check the user has the scm level required for this repository."""
-    if auth0_user.is_in_groups(landing_repo.access_group.active_group):
+    if lando_user.has_perm(
+        f"main.{landing_repo.access_group.membership_group.removeprefix('all_')}"
+    ):
         return None
 
-    if auth0_user.is_in_groups(landing_repo.access_group.membership_group):
-        return "Your {} has expired.".format(landing_repo.access_group.display_name)
-
+    # TODO: update copy to include possibility of expired access?
```

Comment:
If the user doesn't have the Django permission, we should still be able to check `userinfo` directly to see if they are in the `expired_scm_level` group.

Reply:
I have a separate bug filed for this: https://bugzilla.mozilla.org/show_bug.cgi?id=1908702. I think this can be discussed separately from this implementation.